torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -460,6 +462,7 @@ void AliasDb::analyzeImpl(Node* node) {
       return;
     case prim::Print:
     case prim::isinstance:
+    case prim::With:
```

Comment:
As with `prim::Load` and `prim::Store`, `prim::With` is an ir construct that only exists before the IR is in SSA form and valid for Alias Analysis to be called on it. If someone does it should be a hard error, so I think we want to remove this.